draft-ietf-lake-edhoc.md

```diff
@@ -389,54 +392,42 @@ Connection identifiers may be used to correlate EDHOC messages and facilitate th
 
 ## Authentication Parameters {#auth-key-id}
 
-EDHOC supports various settings for how the other endpoint's authentication (public) key is transported, identified, and trusted as described in this section.
+EDHOC supports various settings for how the other endpoint's authentication (public) key may be transported, identified, and trusted.
 
-The authentication key (see {{auth-keys}}) is used in several parts of EDHOC:
+EDHOC performs the following authentication related operations:
 
-1. as part of the authentication credential included in the integrity calculation
-2. for verification of the Signature_or_MAC field in message_2 and message_3 (see {{asym-msg2-proc}} and {{asym-msg3-proc}})
-3. in the key derivation (in case of a static Diffie-Hellman key, see {{key-der}}).
+* EDHOC transports information about credentials in ID_CRED_I and ID_CRED_R (described in {{id_cred}}). Based on this information, the authentication credentials CRED_I and CRED_R (described in {{auth-cred}}) can be obtained. EDHOC may also transport certain authentication related information as External Authorization Data (see {{AD}}).
+* EDHOC uses the authentication credentials in two ways (see {{asym-msg2-proc}} and {{asym-msg3-proc}}):
+    * The authentication credential is input to the integrity verification using the MAC fields.
+    * The authentication key of the authentication credential is used with the Signature_or_MAC field to verify proof-of-possession of the private key.
 
-The authentication credential (CRED_x) contains, in addition to the authentication key, also the authentication key algorithm and optionally other parameters such as identity, key usage, expiry, issuer, etc. (see {{auth-cred}}). Identical authentication credentials need to be established in both endpoints to be able to verify integrity, see {{fig-sigma}}. For many settings it is not necessary to transport the authentication credential within EDHOC over constrained links, for example, it may be pre-provisioned or acquired out-of-band over less constrained links.
+Other authentication related verifications are out of scope for EDHOC, and is the responsibility of the application. In particular, the authentication credential needs to be validated in the context of the connection for which EDHOC is used, see {{auth-validation}}. EDHOC MUST allow the application to read received information about credential (ID_CRED_R, ID_CRED_I). EDHOC MUST have access to the authentication key and the authentication credential.
```

Comment:
Appendix D contains the information I was missing when going through this paragraph. However, the sentence referencing Appendix D is a bit obscure and did not give me an incentive to go read it. Can we rephrase as follows:

s/see {{auth-validation}}/.For examples, see {{auth-validation}.